src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -448,6 +453,71 @@ public void should_create_error_if_type_in_incorrect() {
             .statusCode(404);
     }
 
+    @DisplayName("Valid event id and invalid journey classification")
+    @Test
+    public void should_fail_when_exception_record_has_invalid_journey_classification() throws Exception {
```

Comment:
which one is the correct name?
"Valid event id and invalid journey classification"
or
"Should fail when exception record has invalid journey classification"?

If test name is descriptive I don't see a point in adding annotation with a different name.

Reply:
can remove display name if it is confusing but actually test method needs update.